keyboards/dztech/dz65rgb/v3/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = lufa-dfu
```

Comment:
Is this correct?
From what I can tell the v3 uses the lufa-ms bootloader.
Not the DFU one.

Reply:
See the discussion above.

Reply:
@fauxpark are you sure about that?
It seems that the newer versions of the Lufa-MS bootloader fixed those issues.
I can't say that I'm 100% sure but so far with the stuff that I tested it seems to work fine.

Files that are too large will result in a "no space left on device" error and random non-firmware files dropped into the flashdrive are ignored (and disappear when putting the device back into bootloader mode again).

Reply:
If there is a change of the bootloader in the future, we desided to use  default bootloader, then use hex2bin to change the firmware file format.

Reply:
I tried with a firmware file that QMK's size check said: `26248/28672 (91%, 2424 bytes free)`
It didn't complain that the file was too big - it should have though as latest LUFA has increased the aux bootloader section to 4kB from 2kB (!). Instead it appears to have simply cut off the file after byte 0x5FFF, which sounds about right as 32768 - 0x5FFF = 8192 for the 4kB boot section + 4kB aux.

So no, I still do not recommend using the LUFA mass storage bootloader.

Reply:
Tested this again with a firmware file that used 98% of the flash.
The bootloader does indeed not show any error message that the firmware file is too big and the board does not start afterwards but it also does not seem to brick the device.
Pressing the reset button puts it back into bootloader mode just fine and lets you replace the oversized firmware file with a correct one.

Reply:
`BOOTLOADER_SIZE` will need to be updated, that would hopefully reduce the occurrences of oversized firmware flashing attempts, as the size check should catch it.